### DISCUSSION TITLE
Question about supports

### DISCUSSION THREAD

**Discussion by mitchb2** [OrcaSlicer/OrcaSlicer#3182] [Q&A]
When I slice a model that has supports, I'm confused.  In the preview I see a solid layer, then on the next layer supports only, then on the next layer solid model layer and no supports, etc.  
![layer 3](https://github.com/SoftFever/OrcaSlicer/assets/25935820/aec4ad29-191a-462e-baf9-23265269f159)
![layer 2](https://github.com/SoftFever/OrcaSlicer/assets/25935820/b5df2356-e1dd-4552-abbf-3d4ea7bdfb43)

**Comment by andronick83** [comment]
If I understood the question correctly.
In "Infill combination" mode, support and infill can be printed through a double-height layer.
![image](https://github.com/SoftFever/OrcaSlicer/assets/439962/67e4504f-f4b4-430d-b14a-2e1da5e4f9aa)
